core/frontend/src/components/app/SystemCheckerTrayMenu.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <v-menu
+    v-if="visible"
+    v-model="show_menu"
+    :close-on-content-click="false"
+    nudge-left="200"
+    nudge-bottom="25"
+  >
+    <template
+      #activator="{ on, attrs }"
+    >
+      <v-card
+        class="px-1"
+        elevation="0"
+        color="transparent"
+        v-bind="attrs"
+        v-on="on"
+      >
+        <v-icon
+          class="blinking"
+        >
+          mdi-alert
+        </v-icon>
+      </v-card>
+    </template>
+
+    <v-dialog v-model="show_menu" max-width="700px">
+      <v-card class="text-md-center">
+        <v-card-title>
+          Problems found
+        </v-card-title>
+        <v-card-text>
+          <v-list>
+            <v-list-item v-for="result in results" :key="result.human_message">
+              <v-list-item-content>
+                <v-list-item-title class="text-wrap">
+                  {{ result.human_message }}
+                </v-list-item-title>
+                <v-list-item-subtitle class="text-wrap">
+                  {{ result.debug_message }}
+                </v-list-item-subtitle>
+              </v-list-item-content>
+            </v-list-item>
+          </v-list>
+        </v-card-text>
+        <v-card-actions class="justify-center pa-2">
+          <v-btn
+            color="primary"
+            @click="show_menu = false"
+          >
+            Close
+          </v-btn>
+        </v-card-actions>
+      </v-card>
+    </v-dialog>
+  </v-menu>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+
+import commander from '@/store/commander'
+
+interface TestOutput {
+  human_message: string,
+  debug_message: string,
+}
+
+export default Vue.extend({
+  name: 'VehicleRebootRequiredTrayMenu',
+  data: () => ({
+    show_menu: false,
+    results: [] as TestOutput[],
+    visible: false,
+  }),
+  async mounted() {
+    const tests = [
+      this.check_host_computer_access(),
+    ]
+    const results = await Promise.all(tests)
+    this.results = results.filter((test) => test !== undefined) as TestOutput[]
+    this.visible = this.results.length !== 0
+    if (this.visible) {
+      console.warn(this.results)
+    }
+  },
+  methods: {
+    async check_host_computer_access(): Promise<undefined | TestOutput> {
+      const output = await commander.commandHost('uname -a')
+      if (output === undefined || output.return_code === 0) {
+        return undefined
+      }
+
+      // From: https://github.com/kevinburke/sshpass/blob/master/main.c#L43
+      let human_message
+      switch (output.return_code) {
+        case 0: return undefined
+        case 1: human_message = 'Invalid arguments were provided'; break
+        case 2: human_message = 'Conflicting arguments were provided'; break
+        case 3: human_message = 'A runtime error occurred'; break
+        case 4: human_message = 'An error occurred while parsing'; break
+        case 5: human_message = 'The password provided is incorrect, it should be "raspberry"'; break
```

Comment:
I think we need a clearer message here, actually letting the user know what the issue was and how it was caused

```suggestion
        case 5: human_message = 'Login failed using the password "raspberry", was the "pi" user password changed? For the time being BlueOS requires the host computer username:password to be pi:raspberry'; break
```

I'm. actually not happy with my wording either, so I'll summon @ES-Alexander for help.

Reply:
We can merge at it's to test the functionality and fix the text later, the important thing here to test is the logic, the text can be fixed before tagging stable.